code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -848,15 +852,18 @@
 /datum/chem_property/positive/disrupting/reaction_mob(mob/M, method=TOUCH, volume, potency)
 	if(!isxeno(M))
 		return
-	var/mob/living/carbon/xenomorph/X = M
-	X.interference += (volume * potency)
+	var/mob/living/carbon/xenomorph/xeno = M
+	if(!HAS_TRAIT(xeno, TRAIT_HIVEMIND_INTERFERENCE))
+		to_chat(xeno, SPAN_XENOHIGHDANGER("Your awareness dims to a small area!"))
```

Comment:
It looks like this still gets spammed if you have immunity. So lets instead do it when its Initialized
```suggestion
```